yaml/boundaries.yaml

```diff
@@ -173,7 +177,7 @@ filters:
     min_zoom: 8
     output:
       <<: [ *output_properties, *dispute_properties ]
-      kind: mz_internal_dispute_mask
+      kind: disputed_reference_line
```

Comment:
Is there a reason to keep this as a separate filter now, versus consolidating it with the filter immediately above?

Reply:
This filter catches mainly border=administrative from what I can tell.  I can combine with the above, but it will basically look like two filters in one: 
`if (it's admin level [2,4,6] AND boundary disputed) OR (it has disputed_by AND (dispute=yes or disputed=yes))`.  
I'm not sure it's any cleaner to combine into one filter ... if you're jazzed about it I can do it though